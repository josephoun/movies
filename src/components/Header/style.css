.logo {
    display: flex;
    padding: 20px;
    font-family: fantasy;
    color: black;
}
