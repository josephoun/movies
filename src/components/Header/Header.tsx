import React from "react";
import "./style.css";
import {Link} from "react-router-dom";

function Header() {
    return (
        <header>
            <Link to={'/'}>
                {/*<img src={logo} className="logo" alt="logo" />*/}
                <div className="logo">Next | Movies</div>
            </Link>
        </header>
    );
}

export default Header;
