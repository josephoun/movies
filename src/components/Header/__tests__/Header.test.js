import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom'
import Header from "../Header";

describe("Header component", () => {

    it("should match snapshot", () => {
        const { container } = render(
            <BrowserRouter>
                <Header />
            </BrowserRouter>,
        );
        expect(container).toMatchSnapshot();
    });
});
