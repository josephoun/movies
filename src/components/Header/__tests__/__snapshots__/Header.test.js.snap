// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<div>
  <header>
    <a
      href="/"
    >
      <div
        class="logo"
      >
        Next | Movies
      </div>
    </a>
  </header>
</div>
`;
